modules/light-clients/08-wasm/types/types_test.go

```diff
@@ -77,11 +82,22 @@ func (suite *TypesTestSuite) SetupWasmWithMockVM() {
 
 	suite.coordinator = ibctesting.NewCoordinator(suite.T(), 1)
 	suite.chainA = suite.coordinator.GetChain(ibctesting.GetChainID(1))
+
+	suite.codeHash = storeWasmCode(suite, wasmtesting.CodeHash)
 }
 
 func (suite *TypesTestSuite) setupWasmWithMockVM() (ibctesting.TestingApp, map[string]json.RawMessage) {
 	suite.mockVM = &wasmtesting.MockWasmEngine{}
 	// TODO: move default functionality required for wasm client testing to the mock VM
+	suite.mockVM.StoreCodeFn = func(code wasmvm.WasmCode) (wasmvm.Checksum, error) {
+		hash := sha256.Sum256(code)
```

Comment:
basically `generateWasmCodeHash` that is currently present in `keeper.go`. I believe it would be best suited someplace in `types` (doesn't access keeper in any way) and exported in tests.